src/popsift/popsift.cu

```diff
@@ -18,7 +18,6 @@ using namespace std;
 
 PopSift::PopSift( const popsift::Config& config )
     : _config( config )
-    , _initialized( false )
 {
```

Comment:
We can call `configure( const popsift::Config& config )` to avoid the duplication.